config/initializers/voyager_helpers.rb

```diff
@@ -1,5 +1,4 @@
 require 'yaml'
-require_relative '../../voyager_helpers/lib/voyager_helpers'
 include VoyagerHelpers::OracleConnection
 
 vh_config = YAML.load_file("#{File.dirname(__FILE__)}/../voyager_helpers.yml")
```

Comment:
I think the best option is 2 + 3: a ruby file that uses environment variables.  The sysadmin doesn't need to edit the file at all, and developers can edit the file locally if that's easier than setting env variables.